routes/info_db_routes.py

```
@@ -1,5 +1,5 @@
 from flask import Blueprint, jsonify, request
-from models.info_db import Info_db
+from models.info_db import User
```

Ruff (0.8.2)

`models.info_db.User` imported but unused

Remove unused import: `models.info_db.User`

(F401)